apps/test/unit/lib/kits/maker/dropletConfigTest.js

```diff
@@ -81,3 +88,71 @@ describe('getBoardEventDropdownForParam', function () {
     });
   });
 });
+
+// TODO (bbuchanan): Replace with more general assertions when we move maker
+// commands back to this kit.
+describe(`timedLoop(ms, callback)`, function () {
+  it('is an exported block with expected configuration', function () {
+    const timedLoopBlocks = blocks.filter(block => block.func === 'timedLoop');
+    expect(timedLoopBlocks).to.have.length(1);
+    const timedLoopBlock = timedLoopBlocks[0];
+    expect(timedLoopBlock.func).to.equal('timedLoop');
+    expect(timedLoopBlock.category).to.equal(MAKER_CATEGORY);
+    expect(timedLoopBlock.paletteParams).to.deep.equal(['ms', 'callback']);
+    expect(timedLoopBlock.params).to.deep.equal(['1000', 'function(exit) {\n  \n}']);
+  });
+
+  it('has a matching export in api.js', function () {
+    expect(api).to.haveOwnProperty('timedLoop');
+    expect(api.timedLoop).to.be.a('function');
+  });
+
+  it('api call passes arguments through to Applab.executeCmd', function () {
+    // Check that API passes arguments through as expected
+    window.Applab = window.Applab || {executeCmd() {}};
+    sinon.stub(Applab, 'executeCmd');
+
+    const ms = 234;
+    const callback = function () {};
+    api.timedLoop(ms, callback);
+    expect(window.Applab.executeCmd).to.have.been.calledWith(null, 'timedLoop', {ms, callback});
+
+    window.Applab.executeCmd.restore();
```

Comment:
Nice catch.  I've extracted this into a helper `stubWindowApplab` that should clean up after itself and maybe be useful elsewhere; PTAL.